spec/steps/claim_details/form_spec.rb

```diff
@@ -105,4 +112,18 @@
       end
     end
   end
-end
+
+  context 'when work_before is no' do
+      let(:work_before) { 'no' }
+
+      context 'when all fields are set' do
+        it { expect(subject).to be_valid }
+      end
+
+      context 'when work_before_date is blank' do
+        let(:work_before_date) { nil }
+
+        it { expect(subject).to be_valid }
+      end
+  end
```

Comment:
so... we don't have a test when the value is `yes`, we are also missing tests for work_after

Reply:
still missing test for when it value is yes here